shared-module/epaperdisplay/EPaperDisplay.c

```diff
@@ -157,7 +157,7 @@ static void send_command_sequence(epaperdisplay_epaperdisplay_obj_t *self,
         uint16_t delay_length_ms = 0;
         if (delay) {
             data_size++;
-            delay_length_ms = *(cmd + 1 + data_size);
+            delay_length_ms = *(cmd + 1 + data_size + self->two_byte_sequence_length);
```

Comment:
Thanks -- I think I understand why the existing code is wrong, but maybe you want to have it as `delay_length_ms = *(data + data_size)` so that it always refers to the next byte after the end of `data`. I think that's clearer overall, and might even result in smaller generated code.

Reply:
We can do `*(data+datasize)`, but then I think we have to move the increment of `data_size` to _after_ reading `delay_length_ms`. I can try this out this evening and update the PR.